src/brands/peapods/lib/PeaConnections.js

```diff
@@ -36,27 +68,145 @@ const PeaConnections = ({ classes, followers, followings, tags, groups }) => {
     },
   ];
 
+  const socialMedias = [
+    {
+      provider: 'twitter',
+      icon: 'fab fa-twitter',
+    },
+    {
+      provider: 'facebook',
+      icon: 'fab fa-facebook-f',
+    },
+    {
+      provider: 'linkedIn',
+      icon: 'fab fa-linkedin',
+    },
+    {
+      provider: 'instagram',
+      icon: 'fab fa-instagram',
+    },
+    {
+      provider: 'meetup',
+      icon: 'fab fa-snapchat',
+    },
+  ];
+
+  const settingsOpen = Boolean(anchorEl);
+  const settingsPopoverId = settingsOpen ? 'settings-popover' : undefined;
+
+  const onConnectionsSetting = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const onSettingsClose = () => {
+    setAnchorEl(null);
+  };
+
+  const onSocialClick = (provider, connect) => {
+    setAnchorEl(null);
+    onLinkSocial(provider, connect);
+  };
+
   return (
     <Grid container>
-      {rows.map(({ title, data }) =>
-        data.length ? (
-          <Grid container direction="column">
-            <Grid item>
-              <Typography className={classes.categoryHeading}>
-                {title}
-              </Typography>
-            </Grid>
-
-            <Grid container spacing={2}>
-              {data.map(item => (
-                <Grid key={item.name} item>
-                  <PeaSocialAvatar {...item} />
+      <Grid container alignItems="center" justify="space-between">
+        <Typography className={classes.heading}>Connections</Typography>
+        {loading ? (
+          <PeaLoadingSpinner size={20} />
+        ) : (
+          <IconButton
+            aria-describedby={settingsPopoverId}
+            onClick={onConnectionsSetting}
+          >
+            <PeaIcon
+              icon={'settings'}
+              color="secondary"
+              bgColor={'white'}
+              size={'small'}
+              shadow={false}
+            />
+          </IconButton>
+        )}
+      </Grid>
+
+      {!loading ? (
```

Comment:
minor: it would be more readable to do `loading ? null : <Comp />`

Reply:
sure, done